cmd/ttn-lw-stack/commands/is_db.go

```diff
@@ -89,10 +91,205 @@ var (
 			return nil
 		},
 	}
+	isDBCleanupCommand = &cobra.Command{
+		Use:   "cleanup",
+		Short: "Cleanup expired entities in the Identity Server database",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			logger.Info("Connecting to Identity Server database...")
+			db, err := store.Open(ctx, config.IS.DatabaseURI)
+			if err != nil {
+				return err
+			}
+			defer db.Close()
+			ctx := store.WithSoftDeleted(ctx, true)
+			ctx = store.WithExpired(ctx, config.IS.Delete.Restore)
+			dryRun, err := cmd.Flags().GetBool("dry-run")
+			if err != nil {
+				return err
+			}
+			// find expired applications
```

Comment:
We typically write comments as full sentences, starting with a capital letter and ending with a period.